docs/osm2pgsql.1

```diff
@@ -220,10 +221,10 @@ Instead of storing this information in the main PostgreSQL database, this mode c
 its own separate custom database to store the information. As this custom database
 has application level knowledge about the data to store and is not general purpose,
 it can store the data much more efficiently. Storing the node information for the full
-planet requires about 100GB in PostgreSQL, the same data is stored in only ~16GB using
+planet requires about 300GB in PostgreSQL, the same data is stored in only ~43GB using
 the flat\-nodes mode. This can also increase the speed of applying diff files. This option
 activates the flat\-nodes mode and specifies the location of the database file. It is a
-single large > 16GB file. This mode is only recommended for full planet imports
+single large > 43GB file. This mode is only recommended for full planet imports
```

Comment:
Can we add some description how this is calculated to avoid updating the docs every few months? Maybe "largest node id * 8 bytes". People adding their own fake node ids with higher offsets to their database also need to be aware of flat file space requirements here.

Reply:
@mmd-osm Feel free to write your own PR with more extensive docs.

Reply:
It's a bit of an overkill to a create a pull request for adding `(largest node id * 8 bytes) `. If you don't like to add it, we'll leave it at that.